scripts/toaster-layers-recipes-config.py

```diff
@@ -40,7 +40,7 @@
 
 # Now time to populate custom.xml
 
-for layername in layer_and_recipes:
+for layername in list(layer_and_recipes.keys()):
```

Comment:
.keys() is redundant. Just `list(layer_and_recipes)` will work fine. This looks reasonable, though, you definitely should not be modifying something you're iterating over during the iteration :)